.github/workflows/rws-submissions-checks.yml

```diff
@@ -1,63 +0,0 @@
-# Copyright 2022 Google LLC
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     https://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-name: RWS submission checks
-on: pull_request_target
-jobs:
-  PR-Actions:
-    runs-on: ubuntu-latest
-    outputs:
-      output1: ${{ steps.read_results.outputs.contents }}
-    steps:
-      - name: Get the PR version of the files
-        uses: actions/checkout@v2
-        with:
-          ref: "refs/pull/${{ github.event.number }}/merge"
-          path: "pull-request"
-      - name: Get the version at main
-        uses: actions/checkout@v2
-        with:
-          path: "main"
-      - name: Get necessary libraries
-        run: pip install publicsuffix2
-      - name: Content check
-        id: check
-        run: python3 main/check_sites.py -i pull-request/related_website_sets.JSON --data_directory=main --with_diff --strict_formatting > results.txt
```

Comment:
Looks like the data_directory flag is unused now, so there might be some cleanup to do (unless we have to rework this per my other comment).

Reply:
I think that's a nice feature to have, but it could be removed in a future PR (I want this one to stay focused on just the action).